### DISCUSSION TITLE
Better quality compared to e.g. Gigapixel AI, RRDB_ESRGAN, ESRGAN_PSNR and SRCNN.

### DISCUSSION THREAD

**Discussion by Farathoverfr** [xinntao/Real-ESRGAN#32] [General]
Real-ESRGAN Plus restores fine details and removes compression artifacts very well. I am very impressed with how well your model improves images. I tried it on very highly compressed and small images and was surprised by the high quality.
![2_Original](https://user-images.githubusercontent.com/87935210/129452382-d4d636bd-30e4-49a3-8b7b-d106eb0efa61.png)
![2_Upscale](https://user-images.githubusercontent.com/87935210/129452386-0c3f9512-0834-49e6-a4d8-0650bdf750bd.png)

**Comment by xinntao** [comment]
@Farathoverfr Thanks for your feedback 😄 

Hope we can improve it further :-)

**Reply by Farathoverfr** [reply]
I'm very inspired by what you're doing. I wish you the best of luck. You are very good.

Thank you very much.

**Comment by Ibrahimmughal47** [comment]
Hahahahhahaahahahahaha
